Project-Hotpot/Project-Hotpot/Managers/SpotifyManager.swift

```diff
@@ -134,6 +146,33 @@ extension SpotifyManager: SPTSessionManagerDelegate {
 
 // MARK: - Networking
 extension SpotifyManager {
+    func fetchGenreSeeds(completion: @escaping ([String: Any]?, Error?) -> Void){
+        guard let url = URL(string: "https://api.spotify.com/v1/recommendations/available-genre-seeds"),
+              let accessToken = self.appRemote.connectionParameters.accessToken
+        else {
+            NSLog("accessToken is nil, returning early")
+            return
+        }
+        var request = URLRequest(url: url)
+        request.httpMethod = "GET"
+        let bearer_string = "Bearer " + accessToken
+        request.allHTTPHeaderFields = ["Authorization": bearer_string,
+                                       "Content-Type": "application/json",
+                                       "Accept": "application/json"]
+        //create task
+        let task = URLSession.shared.dataTask(with: request) { data, response, error in
+            guard let data = data,                              // is there data
+                  let response = response as? HTTPURLResponse,  // is there HTTP response
+                  (200 ..< 300) ~= response.statusCode,         // is statusCode 2XX
+                  error == nil else {                           // was there no error, otherwise ...
+                NSLog("Error fetching token \(error?.localizedDescription ?? "")")
+                return completion(nil, error)
+            }
+            let responseObject = try? JSONSerialization.jsonObject(with: data) as? [String: Any]
+            completion(responseObject, nil)
+        }
+        task.resume()
```

Comment:
Relating to my other comment on spotify http code in algirthm class, see how you have repeated code spread across 2 classes whereas you could very easily reuse a private function to get both done in the same class.